tsfc/fem.py

```diff
@@ -697,6 +697,15 @@ def compile_ufl(expression, interior_facet=False, point_sum=False, **kwargs):
 
     # Translate UFL to GEM, lowering finite element specific nodes
     result = map_expr_dags(context.translator, expressions)
+    # Check that any indices from the PointSet have made it into the resulting expression
+    import functools
+    import operator
```

Comment:
Can you move these imports to top-level please?

Reply:
Done in d711e722d19ce86ae5fca8b60e01492bd55e6ae6